content/Store Support/Payments and Invoices/Find-your-receipt-or-invoice.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Find your receipt or invoice"
+id: 360016121859
+---
+
+When you make a purchase with us, your invoice or receipt is sent to the email address you provided at checkout.
+
+## Find your receipt or invoice
+
+Look for an email from us in your inbox. This email is sent after your purchase is processed. The sender details differ depending on whether you purchased from our EU or US store.
+
+**For the EU store:**
```

Comment:
```suggestion
**Europe and Africa:**
```
Used the more customer-facing "region" choice instead of US/EU to distinguish between stores.